site/docs/develop/spec.md

```diff
@@ -158,16 +177,15 @@ definition (think: SQL DDL) or data (think: SQL DML).
 A _Delete operation_ does not carry any Content object and is used to indicate that a Content
 object is no longer referenced using the Content Key of the _Delete operation_.
 
-Example for a Nessie _Delete operation_ is an SQL `DROP TABLE`. An `ALTER TABLE RENAME` is mapped
-to a _Delete operation_ plus a _Put operation_.
-
 ### Unmodified operation
 
 An _Unmodified operation_ does not represent any change of the data, but can be included in a
 Nessie commit operation to enforce strict serializable transactions. The presence of an
 _Unmodified operation_ means that the Content object referred to via the operation's Content Key
 must not have been modified since the Nessie commit's `expectedHash`.
 
+The _Unmodified operation_ is not persisted.
```

Comment:
Would we not need to validate `Unmodifed` on merges?

Reply:
It is validated, but not persisted.

Reply:
How can we validate that `Unchanged` still holds when we merge (if it's not persisted)?